core-primitives/settings/src/lib.rs

```diff
@@ -69,8 +69,11 @@ pub mod worker {
 	pub const SIGNING_KEY_SIZE: usize = 32;
 	// size of the MR enclave
 	pub const MR_ENCLAVE_SIZE: usize = 32;
-	// factor to tune the amount of enclave funding: funding_amount = MIN_FUND_INCREASE_FACTOR*existential_deposit - balance.free
+	// factors to tune the initial amount of enclave funding
+	// min_funding = MIN_FUND_INCREASE_FACTOR*existential_deposit
```

Comment:
It's not the amount to be funded, but the minimal required fund. So minimal required fund ?

Reply:
Is it the minimal required fund? Or is it the maximum amount of funding the enclave receives? I think both are correct, I'd only be glad if you make a clear distinction between the two fundings : both are thresholds, but differ a lot in the way they are used.

Reply:
Both are the minimum fund required. They serve different purposes. 
```
// minimal required fund for development  = MIN_FUND_INCREASE_FACTOR*existential_deposit.
// sufficient to register the enclave and run the worker for a period of time. Only for development

// minimal required fund for production  = MIN_FUND_FACTOR* "enclave registering fee"
// sufficient to register the enclave and start the worker.
```

Reply:
That's a lot better. But because it's not by default "sufficient" but may differ depending on the node, I'd propose to write
`// Should be set to a value that ensures the enclave can register itself and run for a period of time. Only for development.`
Or something in that direction.

And rename the variable names `MIN_FUND_INCREASE_FACTOR` & `MIN_FUND_FACTOR` such that the reprensent what they stand for a little better. Like `FUNDING_AMOUNT_FACTOR` & `MINIMUM_ALLOWED_FUND_FACTOR` .
The word "increase" does not really tell them apart, because both factors increase something.

Reply:
Just a small remark regarding the terms: In English, 'funds' as in 'Mittel', 'Kapital' is always plural. The singular 'fund' is either the verb or has a different meaning.

Reply:
ohh, good to know. Thanks !

Reply:
MINIMUM_ALLOWED_FUNDS_FACTOR * it is then. Though I'm not sure it that's really the best name. Better suggestions are alwas welcomed.

Reply:
I wouldn't make a difference between the 2 factors, except that one is for development only. They have the same meaning: the minimum initial funds required
MIN_DEV_INITIAL_FUNDS_FACTOR because this is only for development
and
MIN_INITIAL_FUNDS_FACTOR

Reply:
So to give my two cents: I think it's the product that is similar, namely the minimum required funds. But the factors themselves represent different things, because one it's applied to the existential deposit and one to the registering fee. So I would include the thing these factors are applied to in the name:
- `EXISTENTIAL_DEPOSIT_INCREASE_FACTOR` 
- `REGISTERING_FEE_INCREASE_FACTOR` 

Although I'm not too sold on the `increase factor` part, but I'd put in the name of the thing that these factors are applied to.

Reply:
I can agree with that. That clearly puts a distinction between the two, whatever they may be used for.

Reply:
Long names? What about:
- EXISTENTIAL_DEPOSIT_FACTOR_FOR_INIT_FUNDS
- REGISTERING_FEE_FACTOR_FOR_INIT_FUNDS

Reply:
Yes, fine for me 👍

Reply:
Yes, I agree as well! Thanks for putting up with us :)